src/ORM/Table.php

```diff
@@ -371,11 +371,17 @@ public function alias($alias = null)
     /**
      * Alias a field with the table's current alias.
      *
+     * If field is already aliased it will result in no-op.
+     *
      * @param string $field The field to alias.
      * @return string The field prefixed with the table alias.
      */
     public function aliasField($field)
     {
+        if (strpos($field, '.') !== false) {
```

Comment:
This won't work well with schema prefixed column names. You could do `strpos($field, $alias) !== false`